http/server.go

```diff
@@ -306,6 +306,7 @@ func NewServer(host string, serviceType common.ServiceType, addr string, port in
 	router.PathPrefix("/statics").HandlerFunc(server.serveStatics)
 	router.PathPrefix(ExtraAssetPrefix).HandlerFunc(server.serveStatics)
 	router.HandleFunc("/login", server.serveLogin)
+	router.PathPrefix("/topology").HandlerFunc(server.serveIndex)
```

Comment:
I would prefer to not add an endpoint right now as not used by the backend

Reply:
I added this as else working in VueRouter(mode: 'history') does not work as explained:

(https://router.vuejs.org/en/essentials/history-mode.html)[history-mode]